### PR TITLE
[fix bug 1194261] Remove old dev-edition /firstrun template

### PR DESCRIPTION
**Pull request by alexgibson**
We removed the waffle switch in #3197, so this older template is now effectively orphaned.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/dev-firstrun.html [hunk 1]
```diff
@@ -77,35 +77,53 @@
       <p>{{_('Get to know the features that make it the most complete browser for building the Web.')}}</p>
     </header>
     <ul class="features">
+    {% if l10n_has_tag('spring_release') %}
       <li class="feature">
-        <a href="https://www.youtube.com/watch?v=1R9_WdXwUsE" rel="external" class="video-play">
-          <img src="{{ static('img/firefox/firstrun/dev/feature-webide.jpg') }}" alt="{{ _('Screenshot') }}" class="screenshot">
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/dev-firstrun.html]: Can delete this image?

File: bedrock/firefox/templates/firefox/dev-firstrun.html [hunk 2]
```diff
@@ -77,35 +77,53 @@
...
+        {% endif %}
         <div class="responsive-video-container">
-          <div class="video" data-video-id="1R9_WdXwUsE"></div>
+          <div class="video" data-video-id="eH0R10Ga4Hs"></div>
         </div>
       </li>
+    {% endif %}
       <li class="feature">
-        <a href="https://www.youtube.com/watch?v=eH0R10Ga4Hs" rel="external" class="video-play">
-          <img src="{{ static('img/firefox/firstrun/dev/feature-valence.jpg') }}" alt="{{ _('Screenshot') }}" class="screenshot">
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/dev-firstrun.html]: And this one?

File: bedrock/firefox/templates/firefox/dev-firstrun.html [hunk 3]
```diff
@@ -117,24 +135,26 @@
         <h2>{{_('Features and tools')}}</h2>
       </header>
       <ul class="features">
+      {% if l10n_has_tag('spring_release') %}
         <li class="feature">
-          <a href="https://www.youtube.com/watch?v=eQqNfkqIJdw" rel="external" class="video-play">
-            <img src="{{ static('img/firefox/firstrun/dev/feature-inspector.jpg') }}" alt="{{ _('Screenshot') }}" class="screenshot">
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/dev-firstrun.html]: And this one?

**Comment by jpetto** [comment]
Other than orphaned images, looks good. r+wc :ribbon:

**Comment by alexgibson** [comment]
Thanks @jpetto - updated and removed orphaned images.

**Pull request merged**